### DISCUSSION TITLE
Is it possible to use the vertical scrolling of the touchpad to zoom in?

### DISCUSSION THREAD

**Discussion by boromyr** [qsniyg/maxurl#1331] [Q&A]
idem

**Comment by qsniyg** [comment]
It should be possible by default. Otherwise, change the "Vertical scroll action" under the Popup tab of the Options page to "Zoom".

**Comment by boromyr** [comment]
Yes indeed it is correctly set to zoom, but it doesn't work for me. If I set it to "pan" vertical or horizontal everything works correctly and the image moves, but if I set it to zoom it's like it doesn't read the vertical movement, while the horizontal one works. I installed the script both as an extension in dev mode and on Tampermonkey, it doesn't work for either. Edge 126
